### PR TITLE
Feature/postcss v8 plugin api

### LINKED ISSUE #1: PostCss v8 Compatibility
If I use fail-on-warn on its own code, I get a warning that the plugin should be migrated with this guide https://evilmartians.com/chronicles/postcss-8-plugin-migration

I would like to submit a PR with an updated plugin api for postcss v8.

### PR DESCRIPTION
**Pull request by sebastianrothe**
Uses PostCss v8 Plugin API via https://evilmartians.com/chronicles/postcss-8-plugin-migration

- updated packages
- formatted files to latest eslint rules

Fixes #1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
-var postcss = require('postcss')
+const warningsReporter = () => {
+  return {
+    postcssPlugin: 'postcss-warnings-reporters',
```
Comment (ai) [inline index.js]:
```suggestion
    postcssPlugin: 'postcss-fail-on-warn',
```

Comment (sebastianrothe) [reply index.js]: Ah sorry, my bad.

**Comment by ai** [comment]
I also improved warning formatting  c96af94

**Comment by ai** [comment]
Released in 0.2

**Pull request merged**